app/views/lessons/index.html.erb

```diff
@@ -2,16 +2,21 @@
 
 <div class="row my-3">
   <h1>Categories (<%= @categories.count %>)</h1>
-  <button class="btn btn-primary ml-auto">All</button>
-  <button class="btn btn-primary mx-3">Learned</button>
-  <button class="btn btn-primary">Not learned</button>
+  <%= link_to "All" , lessons_path, class: "btn btn-primary ml-auto" %>
+  <%= link_to "Learned" , lessons_path(filter: "Learned"), class: "btn btn-primary mx-3" %>
+  <%= link_to "Not Learned" , lessons_path(filter: "Not Learned"), class: "btn btn-primary" %>
```

Comment:
to fix button size you can write it like this:
```
  <div class="ml-auto">
    <%= link_to "All" , lessons_path, class: "btn btn-primary" %>
    <%= link_to "Learned" , lessons_path(filter: "Learned"), class: "btn btn-primary mx-3" %>
    <%= link_to "Not Learned" , lessons_path(filter: "Not Learned"), class: "btn btn-primary" %>
  </div>
```
![image](https://user-images.githubusercontent.com/73863408/142356959-a99edc2e-7d66-4629-afde-56daed9bf5c0.png)